README.md

```diff
@@ -43,6 +43,21 @@ $ disl your-image-name:label 300MiB --max-layers=5
 # ok!
 ```
 
+Add `--current` or `-c` flag to show the current size your image:
```

Comment:
```suggestion
Add `--current-size` flag to show the current size your image:
```

Reply:
Done in https://github.com/wemake-services/docker-image-size-limit/commit/b76c142f89ffe291e1c800b1daa2c6ebf46a2e4f